zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/message/DbProcessMessageSubscriptionState.java

```diff
@@ -171,7 +174,12 @@ public void visitPending(final long deadline, final ProcessMessageSubscriptionVi
               pendingSubscription.elementInstanceKey(),
               BufferUtil.wrapString(pendingSubscription.messageName()),
               pendingSubscription.tenantId());
-
+      if (subscription == null) {
+        LOG.warn(
+            "Expected to find subscription with key {}, but no subscription found",
+            pendingSubscription.elementInstanceKey());
```

Comment:
❌ actually all parameters are used for lookup, let's include them in the log to be able to reconstruct the whole key
```suggestion
            "Expected to find subscription with key {} messageName {} tenantId: {}, but no subscription found",
            pendingSubscription.elementInstanceKey(), messageName, tenantId);
```